src/Storage.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Biigle\Modules\Kpis;
+
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+
+class Storage
+{
+    public static function getStorageUsage($year, $month)
+    {
+        $date = Carbon::createFromDate($year, $month, 1)->endOfMonth()->toDateString();
+        $res = count(DB::table('kpis_storage_usage')->where('date', '=', $date)->pluck('value')) != 0 ?
+        DB::table('kpis_storage_usage')->where('date', '=', $date)->pluck('value')[0] : 0;
```

Comment:
While it's always just one row, `sum()` is convenient and returns 0 if no row exists.
```suggestion
        $res = DB::table('kpis_storage_usage')->where('date', '=', $date)->sum('value');
```